configs/HWH7/config.h

```diff
@@ -0,0 +1,213 @@
+/*
+- This file is part of Betaflight.
+*
+- Betaflight is free software. You can redistribute this software
+- and/or modify this software under the terms of the GNU General
+- Public License as published by the Free Software Foundation,
+- either version 3 of the License, or (at your option) any later
+- version.
+*
+- Betaflight is distributed in the hope that it will be useful,
+- but WITHOUT ANY WARRANTY; without even the implied warranty of
+- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*
+- See the GNU General Public License for more details.
+*
+- You should have received a copy of the GNU General Public
+- License along with this software.
+*
+- If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32H743
+#define BOARD_NAME        HWH7
+#define MANUFACTURER_ID   HOWI
+
+//Gyroscope
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
```

Comment:
The schematic uses ICM42688P. You cannot use MPU6000 on the same footprint as they have different orientations. 
```suggestion
#define USE_ACC_SPI_ICM42688P

#define USE_GYRO
```

Reply:
Resolved